addons/mrp/models/mrp_production.py

```diff
@@ -803,6 +803,20 @@ def unlink(self):
             workorders_to_delete.unlink()
         return super(MrpProduction, self).unlink()
 
+    def copy(self, default=None):
+        new_mo = super(MrpProduction, self).copy(default=default)
+        # cover 2 cases: backorders generation (follow default logic for moves copying)
+        # and copying a done MO via the form (i.e. copy only the non-cancelled moves since no backorder = cancelled finished moves)
+        if not default or default.get('move_finished_ids', False) is not None:
```

Comment:
`default.get('move_finished_ids', False) is not None`, if you send a {'move_finished_ids': False}, you will copy move_finished_ids :thinking: 
Maybe `if not default or 'move_finished_ids' not in default`

Reply:
Agreed, changed it